### PR TITLE
documentation improvements

### PR DESCRIPTION
**Pull request by docgalaxyblock**
I have worked a bit on the individual wiki sites layout & formatting
Two things to note
- Added a `Get it on F-Droid` Badge
- local asteroidos.org container is now identified by its tag (asteroisos-web) and every commandrefernces that now

everything else are minor edits regarding formatting of inlinecode, codeblocks, etc

### CODE REVIEW

File: pages/wiki/porting-guide.md [hunk 1]
```diff
@@ -20,76 +20,76 @@ layout: documentation
...
+<p>Start by cloning the most appropriate layer into an <code>src/meta-XXX-hybris/</code> directory in your AsteroidOS build tree. From now on, XXX will be a reference to&nbsp;your watch’s codename. Once you have that layer setup, <a href="https://github.com/AsteroidOS/meta-swift-hybris/commit/b665878f80936f21cec1757cdc4d10fa95af1841">roughly&nbsp;replace</a> all references to the previous codename&nbsp;with XXX in the layer and add the path to your layer in your <code>build/conf/bblayers.conf</code> file.</p>
```
Comment (beroset) [inline pages/wiki/porting-guide.md:23]: Now I see why you were asking about the `&nbsp;` on Matrix.  All of these on this page should be deleted.

**Review by beroset — APPROVED** [review]
Definite improvement, thanks!

File: pages/wiki/porting-guide.md (hunk 1 above)
Comment (docgalaxyblock) [reply pages/wiki/porting-guide.md:23]: Ok and the other pages?

Comment (beroset) [reply pages/wiki/porting-guide.md]: A quick check shows that they probably should be removed in all pages.  For just the `useful-commands.md` file, each time it's used, it is just before a `####` heading marker, but the better way to create extra spacing is to omit the `&nbsp;` and use a `###` heading marker instead.

**Review by beroset — APPROVED** [review]

**Comment by FlorentRevest** [comment]
Excellent, thanks a lot!

**Pull request merged**